examples/step-37/step-37.cc

```diff
@@ -1132,13 +1106,22 @@ namespace Step37
 
   // Here is the data output, which is a simplified version of step-5. We use
   // the standard VTU (= compressed VTK) output for each grid produced in the
-  // refinement process. We disable the output when the mesh gets too
-  // large. Note that a variant of program has been run on hundreds of
-  // thousands MPI ranks with as many as 100 billion grid cells, which is not
-  // directly accessible to classical visualization tools.
+  // refinement process. In addition, we use a compression algorithm that is
+  // optimized for speed rather than disk usage. The default setting (which
+  // optimizes for disk usage) makes saving the output take about 4 times as
+  // long as running the linear solver, while setting
+  // DataOutBase::VtkFlags::compression_level to
+  // DataOutBase::VtkFlags::best_speed lowers this to only one fourth the time
+  // of the linear solve.
+  //
+  // We disable the output when the mesh gets too large. Note that a variant
+  // of program has been run on hundreds of thousands MPI ranks with as many
```

Comment:
This is pre-existing, but would you rephrase "a variant of this program has been run..."